packages/types/src/config-schema.json

```diff
@@ -988,127 +988,6 @@
       },
       "required": ["operations"]
     },
-    "JsonSchemaHTTPOperation": {
```

Comment:
no sure the changes below are expected 👀

Reply:
I think it is just missing `yarn build` call.